src/btree.c

```diff
@@ -3083,13 +3083,66 @@ btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_btree_pivot_stats(platform_log_handle *log_handle,
+                              btree_pivot_stats   *pivot_stats)
+{
+   // If nothing has been updated, yet, print a msg accordingly.
+   if ((pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->key_bytes == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->message_bytes == BTREE_UNKNOWN_COUNTER))
+   {
+      platform_log(log_handle, "   (Pivot Stats=BTREE_UNKNOWN_COUNTER)\n");
+      return;
+   }
+
+   // Indentation is dictated by outer caller
+   size_t nkbytes = pivot_stats->key_bytes;
+   char   nkbytes_str[SIZE_TO_STR_LEN];
+
+   size_t nmbytes = pivot_stats->message_bytes;
+   char   nmbytes_str[SIZE_TO_STR_LEN];
+   platform_log(log_handle,
+                "   (num_kvs=%u\n"
+                "    key_bytes=%lu (%s)\n"
+                "    message_bytes=%lu (%s))\n",
+                pivot_stats->num_kvs,
+                nkbytes,
+                size_to_str(nkbytes_str, sizeof(nkbytes_str), nkbytes),
```

Comment:
The newly added interface to print with unit-specifiers is applied here to print BTree pivot stats fields. The new output will look like follows, as generated by `./build/release/bin/unit/btree_stress_test`, which does a `btree_pack()`, which will set these values:

```
BTree node at addr=1333133312
{
**  INDEX NODE
**  Header ptr: 0x7f55c36ce000
**  addr: 1333133312
**  next_addr: 0
[...]
-------------------
Offset Table num_entries=3
[0] 4010    [1] 3884    [2] 3787
-------------------
Array of 3 index entries:
[ 0]: key=0x00000d093a9a17000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
   child_addr=1333657600
   (num_kvs=377607
    key_bytes=21716772 (~20.71 MiB)
    message_bytes=260863299 (~248.77 MiB))
[ 1]: key=0x60aaf36fb8dd4d000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
   child_addr=1333661696
   (num_kvs=379751
    key_bytes=21839065 (~20.82 MiB)                        <<< You will now get the bytes + string with unit-specifiers.
    message_bytes=262304591 (~250.15 MiB))        <<<<
...
`